### ISSUE TITLE
No longer respecting REDACT on client. Passwords are shown in check

### ISSUE THREAD

**Issue by zbintliff** [sensu/sensu#1333]
With this commit:
https://github.com/sensu/sensu/commit/1917db8ff20a5ef6f40aa490f7a5658ae1253af5#diff-126205c86cd89c693bea304046d9006e

Since you now substitute on the check itself instead on a seperate `command` string. the check is modified. When you publish the check any passwords are now in raw text.  The redact is respected in the client hash but not the check page.

For example, before the check the check result would like so in uchiwa
```
command: /usr/bin/check-db.rb -h :::host|default::: -p :::password|default:::
```

After change it looks like:
```
command: /usr/bin/check-db.rb -h real_hostname -p real_password
```

this is a pretty big vuln where passwords need to be redacted.

**Comment by zbintliff** [comment]
thanks for the fast response!  I saw this was merged into 0.25, was wondering if its worth putting in 0.24.2?

**Comment by portertech** [comment]
@zbintliff 0.25 is a small (but important) release, https://github.com/sensu/sensu/issues?q=is%3Aissue+milestone%3A0.25+is%3Aclosed
